### ISSUE TITLE
Provide jump link for search on mobile

### ISSUE THREAD

**Issue by anselmbradford** [codeforamerica/ohana-web-search#168]
Search appears below results list on mobile (#151) but should have a fixed link to jump to the search box so the user does not need to scroll through all results to get to the search box again after performing a search.
